discovery/ionos/server_test.go

```diff
@@ -61,7 +61,7 @@ func TestIONOSServerRefresh(t *testing.T) {
 			"__meta_ionos_server_availability_zone": "ZONE_2",
 			"__meta_ionos_server_boot_cdrom_id":     "0e4d57f9-cd78-11e9-b88c-525400f64d8d",
 			"__meta_ionos_server_cpu_family":        "INTEL_SKYLAKE",
-			"__meta_ionos_server_datacenter_id":     "8feda53f-15f0-447f-badf-ebe32dad2fc0",
+			"__meta_ionos_server_datacenter_id":     "8feda53f-15f0-447f-badf-ebe32dad2fc0/servers",
```

Comment:
__meta_ionos_server_servers_id

Reply:
I've updated the label, thanks for the catch.